routes/web.php

```diff
@@ -25,7 +25,8 @@
 Route::get('/create-course', 'App\Http\Controllers\CreateCourse@index')->name('create-courses');
 
 
-
+ 
+Route::get('/vicelab-exp2','App\Http\Controllers\experimentController@vicelabexp2')->name('vicelab-exp2');
```

Comment:
@saidabdul80 please use a more descriptive url path. not sure `/vicelab-exp2` looks good like the others.

Reply:
Ok